freqtrade/tests/strategy/test_strategy.py

```diff
@@ -26,6 +26,15 @@ def test_load_strategy(result):
     assert 'adx' in resolver.strategy.populate_indicators(result)
 
 
+def test_load_strategy_from_url(result):
+    resolver = StrategyResolver()
+    resolver._load_strategy('https://raw.githubusercontent.com/berlinguyinca'
```

Comment:
This test will fail if no network is present ... i think this should be mocked to not require network connectivity

Reply:
good call. Somehow I always take a network in granted in 2018 :)